<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>107</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>91</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8888</width>
    <height>8888</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>664</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget_inPath">
       <property name="minimumSize">
        <size>
         <width>411</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_chooseFile">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBox_check">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
